sapientml_preprocess/generator.py

```diff
@@ -31,7 +32,7 @@
 
 logger = setup_logger()
 
-INHIBITED_SYMBOL_PATTERN = re.compile(r"[\{\}\[\]\",:<'\\]+")
+INHIBITED_SYMBOL_PATTERN = re.compile(r"[\{\}\[\]\",:<'\\\+]+")
```

Comment:
Why did you add `\+`?

Reply:
I was checking if "+" is treated as a symbol in the test string.
I put it back.